### DISCUSSION TITLE
New .name.php files are created

### DISCUSSION THREAD

**Discussion by DigiRon4ik** [yandeu/five-server#30] [Help / Questions] [answered]
Hi!

Briefly speaking. The problem is this. Only when I use your **Five Server** do I have when editing any _.php_, _.html_, etc. file. copies of these files are created only with a dot at the beginning:

**Original:** _index.php_
**Copy:** _.index.php_

And this is not even a hidden file, but a full-fledged one, if I try to edit _.index.php_ already, _..index.php_ will be created, and it infuriates.
As already said, this is only with **Five Server**, when using **PHP Server**, this is not.

https://user-images.githubusercontent.com/84411333/141657102-98806d73-d47a-4bfa-86c9-0d79877a34e3.mp4

_If anything, I'm Russian._

**Comment by DigiRon4ik** [comment]
I found an approximate solution to this, you need to disable injectBody, but after that there is no selection on the page, although in your video **"Five Server: PHP (instant update)"** everything works fine and does not create copies.

**Comment by yandeu** [comment]
I will see if I can improve it.

---

For now:

In VSCode you could add a `settings.json` file inside a `.vscode` folder, to hide all `.*.php` files.

```json
// .vscode/settings.json
{
  "files.exclude": {
    "www/**/.*.php": true
  }
}
```

**Reply by DigiRon4ik** [reply]
Is there an option to somehow change the creation path of these files, so that they are created in the _.vscode_ folder, and also add the _"hidden"_ tag to it to hide it in the explorer?

Or delete these files after **Five Server** is finished?

**Comment by yandeu** [comment, marked as answer]
Has been fixed in the latest version.

**Comment by DigiRon4ik** [comment]
thanks
